CMakeLists.txt

```diff
@@ -97,6 +97,10 @@ if(DL_KERNELS)
     add_definitions(-DDL_KERNELS)
     set(CK_ENABLE_DL_KERNELS "ON")
 endif()
+if(CK_USE_CODEGEN)
```

Comment:
You should do `option(CK_USE_CODEGEN "Enable codegen library" OFF)` before this.